dd-sdk-android-ndk/src/main/kotlin/com/datadog/android/ndk/NdkCrashReportsPlugin.kt

```diff
@@ -61,6 +63,13 @@ class NdkCrashReportsPlugin : DatadogPlugin {
         unregisterSignalHandler()
     }
 
+    override fun onContextChanged(context: RumPluginContext) {
+        super.onContextChanged(context)
+        updateRumContext(context.applicationId, context.sessionId, context.viewId)
```

Comment:
Shouldn't we also check if RUM is enabled and user opted-in for bundling RUM info with Logs? As we do in `DatadogLogHandler.kt`:
https://github.com/DataDog/dd-sdk-android/blob/4b2234f97c01629ac6fbbf03def62be45638f150/dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/logger/DatadogLogHandler.kt#L77-L82

Reply:
Yeah...first part is by default otherwise you would not get updates for the RUM context. Second part I was aware of but thought maybe for the crashes we should never condition that. For the current JVM crash reporter we are not doing this at all so I opened a ticket there and after we fix that and we decide if we should condition that also on the `bundleWithRum` parameter I will try to see how I can add this in the NDK reporter also.

Reply:
Sounds good 👌, I just wanted to draw the attention to this.

Reply:
No worries...my bad I did not left a comment. I will actually create a PR with a TODO.